plugins/org.locationtech.udig.tutorials.rcp/src/org/locationtech/udig/tutorials/rcp/OverviewMapView.java

```diff
@@ -245,6 +245,22 @@ public void run() {
 
     }
 
+    @Override
+    public UDIGDropHandler getDropHandler() {
+        return mapviewer.getDropHandler(); // oder overviewmapviewer.getDropHandler() ????
+    }
+
+    // helper methods for Toolmanager
+    @Override
```

Comment:
You may use the /** comment instead of // ?

Reply:
just remove `// helper methods for Toolmanager`